cost/budget_alerts/budget_alert.pt

```diff
@@ -257,10 +257,12 @@ end
 
 script "js_filtered_billing_centers", type: "javascript" do
   parameters "billing_centers", "param_bc_name"
-  result "filtered_billing_centers"
+  result "filtered_billing_centers_new"
   code <<-EOS
   var filtered_billing_centers =
     _.reject(billing_centers, function(bc){ return (param_bc_name !== bc.name) });
+  var filtered_billing_centers_new =
+    _.reject(filtered_billing_centers, function(bc){ return (bc.parent_id != null) })
```

Comment:
I'm not sure if this is what we want.  Can you explain the logic?  It would be helpful to include some comments about what the new line is doing for anybody reading the code.  Also do you get the same results of the billing centers that are not Unallocated?

Reply:
Hi @cdwilhelm , This new line is filtering out the child BC and we are left with only the Parent BC. As in case of unallocated BC, the child Unallocated BC will be filtered out. I will add the comment in the policy.